### ISSUE TITLE
Firefox 4b1: Annoying menu cursor lag

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/quake2-gwt-port#29]
```
What steps will reproduce the problem?
1. Enter the menu
2. Move the cursor

What is the expected output? What do you see instead?

Cursor movement should be nice and fast.  However the browser seems to 
noticeably lag every time you move the cursor.  It's easiest to see on the main 
menu where the cursor animates that there is indeed a lag caused by simply 
moving the cursor.

I am uncertain if there was a similar delay in Chromium (since it's so much 
faster).  But I believe there is, albeit a much smaller delay.
```

---

Original issue reported on code.google.com by `megazzt` on 23 Jul 2010 at 10:57

**Comment by GoogleCodeExporter** [comment]
```
It's no secret that Firefox 4.0 beta 1 was relatively slow in the JS 
department, this will be fixed by the time 4.0 final is released (actually the 
upcoming beta4 should already be much faster). In any case this is not a 
problem in quake2-gwt, no need to report here.
```

---

Original comment by `jacob.be...@gmail.com` on 18 Aug 2010 at 7:25

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `stefan.haustein` on 25 Aug 2010 at 12:38
- Changed state: **Invalid**
